### PR TITLE
configuring multiple host

### PR DESCRIPTION
**Pull request by bibi-da**

### CODE REVIEW

File: lib/sevak/core.rb [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
+        host = config.host.kind_of?(Array) ? config.host : [config.host]
+        @conn ||= Bunny.new(
+          hosts: host,
+          port: config.port,
+          username: config.user,
+          password: config.password)
+        @conn.start
+        @conn
+      rescue Bunny::TCPConnectionFailedForAllHosts => e
+        sleep(0.0001)
```
Comment (kmaheshwari) [inline lib/sevak/core.rb]: Isn't this sleep time too less? In case if a node is down this will retry 1000 times/s

Comment (deepakkumarnd) [reply lib/sevak/core.rb]: I think the sleep time can be increased This 0.1 ms is too very less.

File: lib/sevak/core.rb [hunk 2]
```diff
@@ -1,15 +1,23 @@
...
+        retry
+      rescue Bunny::TCPConnectionFailed => e
+        sleep(0.0001)
```
Comment (deepakkumarnd) [inline lib/sevak/core.rb]: Possibility of a loop and we won't know the issue

File: lib/sevak/core.rb [hunk 3]
```diff
@@ -1,15 +1,27 @@
...
+            hosts: host,
+            port: config.port,
+            username: config.user,
+            password: config.password)
+          @conn.start
+          @conn
+        end
+      rescue Bunny::TCPConnectionFailedForAllHosts => e
+        attempt += 1
+        sleep(0.1)
```
Comment (deepakkumarnd) [inline lib/sevak/core.rb]: sleep(0.1) if attempt < 10 would be neat, or the main bock will be a a conditional

File: lib/sevak/core.rb [hunk 4]
```diff
@@ -1,15 +1,27 @@
...
+        retry
+      rescue Bunny::TCPConnectionFailed => e
+        attempt += 1
+        sleep(0.1)
```
Comment (deepakkumarnd) [inline lib/sevak/core.rb]: here too

**Review by deepakkumarnd — APPROVED** [review]

**Pull request merged**